README.md

```diff
@@ -563,6 +565,44 @@ Brainstem provides a rich DSL for building presenters.  This section details the
     association :associated_objects, :polymorphic, "a mixture of objects related to this Widget"
   end
   ```
+  
+* `lookup` - Use this option to avoid N + 1 queries for Fields and Associations. The `lookup` lambda runs once when
+presenting and every presented model gets its assocation or value from the cache the `lookup` lambda generates. The
+`lookup` lambda takes in the presented models and should generate a cache containing the models' coresponding assocations
+or values. Brainstem expects the return result of the `lookup` to be a Hash where the keys are the presented models' ids
```

Comment:
Use this in error: `Brainstem expects the return result of the `lookup` to be a Hash`